### ISSUE TITLE
Accessible names for permalinks

### ISSUE THREAD

**Issue by mattgarrish** [w3c/respec#3853]
## Important info

- URL to affected spec: https://w3c.github.io/epub-specs/epub33/core/
- ReSpec version: 26.16.1
- [x] I did a "hard refresh", but it's still busted.

## Description of problem

Another respec-related issue that came up in the epub accessibility horizontal review was the lack of meaningful labels for the permalinks. Currently they all get an `aria-label` attribute with the value '§'.

I've patched this for epub in a post-processing step that identifies what is getting permalinked and using that label with the number. For example, "Permalink for Section #", "Permalink for Example #", etc.

It looks like this problem was manually fixed in the last WCAG release to use the entire heading value, so you get "Permalink for #.#.# Blah blah title".

That looks simpler to implement, but I'm not sure if it would be annoying to users given its repetition of the entire title. We'll need to figure out what approach is best before a solution can be proposed.

For more info about the original issue, see https://github.com/w3c/epub-specs/issues/1827

**Comment by marcoscaceres** [comment]
I'm open to suggestions... we could go with "Permalink for #" and see if anyone complains?

**Comment by mattgarrish** [comment]
> we could go with "Permalink for #" and see if anyone complains?

Ya, that should work.

I think I've been overthinking the problem here, as it's only sections/appendixes that have the permalinks separate from their titles. Examples and WebIDL use their titles as the permalinks, so adding on aria-label would be redundant.
